resources/web/wwi/FloatingWindow.js

```diff
@@ -64,21 +92,21 @@ export default class FloatingWindow {
     return this.floatingWindow.style.visibility;
   }
 
-  dragElement(fw) {
+  _dragElement(fw) {
     let pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
-    let pageHeight = fw.parentNode.parentNode.parentNode.offsetHeight;
-    let containerHeight = fw.parentNode.parentNode.offsetHeight;
-    let containerWidth = fw.parentNode.parentNode.offsetWidth;
+    let pageHeight = fw.parentNode.parentNode.offsetHeight;
+    let containerHeight = fw.parentNode.offsetHeight;
+    let containerWidth = fw.parentNode.offsetWidth;
```

Comment:
const?

Reply:
These values will change if the user changes screen size. I just tested with const and it doesn't work.